src/lammpsio/topology.py

```diff
@@ -194,14 +213,39 @@ def reorder(self, order, check_order=True):
 
 
 class Bonds(Topology):
-    """Particle bonds.
+    """Bond connections between particles.
+
+    All values of indexes follow the LAMMPS 1-indexed convention, but the
+    arrays themselves are 0-indexed.
 
     Parameters
     ----------
     N : int
         Number of bonds.
     num_types : int
-        Number of bond types.
+        Number of bond types. Default of ``None`` means the number of types is
+        determined from the unique typeids.
+
+    Example
+    -------
+    Create bonds:
+
+    .. code-block:: python
+
+        snapshot = lammpsio.Snapshot(
+            N=3,
+            box=lammpsio.Box([-5, -5, -5], [5, 5, 5]),
+            step=10
+        )
```

Comment:
Could this be an invisible part of the code so that these run in Sybil but this example strictly focuses on topology and just assumes there is an existing snapshot that is being added to? Same for below, if so